arch/cortex-m/src/systick.rs

```diff
@@ -193,9 +198,9 @@ impl kernel::platform::scheduler_timer::SchedulerTimer for SysTick {
             ControlAndStatus::CLKSOURCE::SET
         };
 
-        // We really just need to set the TICKINT bit here, but can't use modify() because
-        // readying the CSR register will throw away evidence of expiration if one
-        // occurred, so we re-write entire value instead.
+        // We really just need to set the TICKINT bit here, but can't use modify
+        // () because readying the CSR register will throw away evidence of
+        // expiration if one occurred, so we re-write entire value instead.
```

Comment:
```suggestion
        // We really just need to set the TICKINT bit here, but can't `modify()`
        // because readying the CSR register will throw away evidence of
        // expiration if one occurred, so we re-write entire value instead.
```